tests/bluemira/geometry/test_parameterisations.py

```diff
@@ -27,14 +27,21 @@
 import numpy as np
 import pytest
 
+from bluemira.display import plot_2d
```

Comment:
We try not to plot in tests any more. I appreciate they're useful, but it makes the tests a pain to run. Remove this import and the plot calls